### PR TITLE
update readme doc

### PR DESCRIPTION
**Pull request by calvin0327**
Signed-off-by: calvin0327 <wen.chen@daocloud.io>

/kind doc

add `how to set yourself cloudshell images paragraph` to readme.

### CODE REVIEW

**Comment by calvin0327** [comment]
@panpan0000 /cc

File: README.md [hunk 1]
```diff
@@ -82,6 +82,16 @@ After the cloudtty is intergated to your own UI, it would look like:
 
   ![screenshot_png](https://github.com/cloudtty/cloudtty/raw/main/docs/snapshot.png)
 
+### Use custom cloudshell image
+
+Most users need more than just the basic Kubectl tools to manage their clusters. Users can customize images based on cloudshell base images. set the 'JobTemplate' image parameter to run your own cloudshell image when install cloudtty.
```
Comment (panpan0000) [inline README.md:87]: when installing cloudtty.

File: README.md [hunk 2]
```diff
@@ -82,6 +82,16 @@ After the cloudtty is intergated to your own UI, it would look like:
...
+
+```shell
+helm install cloudtty-operator --version 0.3.0 daocloud/cloudtty --set jobTemplate.image.registry=</your/registry> --set jobTemplate.image.repository=</your/repository> --set jobTemplate.image.tag=</your/tag>
```
Comment (panpan0000) [inline README.md:90]: Can you add the ** sample Dockerfile **  to add a tool into the job images as well ?

Comment (calvin0327) [reply README.md:90]: 👌

**Comment by calvin0327** [comment]
@panpan0000 add a new paragraph to introduce how to add constom tool to cloudshell image.
PTAL

File: README.md [hunk 3]
```diff
@@ -82,6 +82,60 @@ After the cloudtty is intergated to your own UI, it would look like:
...
   ![screenshot_png](https://github.com/cloudtty/cloudtty/raw/main/docs/snapshot.png)
 
+### How to build custom cloudshell image
+
+Most users need more than just the basic `kubectl` tools to manage their clusters. we can customize image based on cloudshell base image. here is an example of adding the `karmadactl` tool.
+
+* Modify cloudshell basic ![Dockerfile](https://github.com/cloudtty/cloudtty/blob/main/docker/Dockerfile-webtty).
+
+```shell
+FROM node:18.5.0 as builder
```
Comment (panpan0000) [inline README.md:92]: why not FROM our existing `ghcr.io/cloudtty/cloudshell:v0.3.0` ?

**Pull request merged**